pom.xml

```diff
@@ -36,7 +36,7 @@
         <maven.compiler.target>17</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <spring.boot.version>3.1.4</spring.boot.version>
+        <spring.boot.version>3.1.6</spring.boot.version>
```

Comment:
Why not 3.2.0 used in Flow? See: [vaadin/flow/pom.xml#L87](https://github.com/vaadin/flow/blob/5ca658bf3c8ff610b8741605865f540a1c6b783a/pom.xml#L87)